SPECS/rubygem-power_assert/rubygem-power_assert.spec

```diff
@@ -0,0 +1,51 @@
+%global gem_name power_assert
+Summary:        Power Assert for Ruby
+Name:           rubygem-power_assert
+Version:        2.0.1
+Release:        3%{?dist}
+License:        BSD-2-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://github.com/ruby/power_assert/
+Source0:        https://github.com/ruby/power_assert/archive/refs/tags/v%{version}.tar.gz#/%{gem_name}-%{version}.tar.gz
+Patch0:         remove-missing-devdeps.patch
+BuildRequires:  git
+BuildRequires:  ruby
+Provides:       rubygem(power_assert) = %{version}-%{release}
+BuildArch:      noarch
+
+%description
+Power Assert shows each value of variables and method calls in the expression.
+It is useful for testing, providing which value wasn't correct when the
+condition is not satisfied.
+
+%prep
+%autosetup -p1 -n %{gem_name}-%{version}
+
+%build
+gem build %{gem_name}
+
+%install
+gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{gem_name}-%{version}.gem
+#add BSDL and COPYING files to buildroot from Source0
+cp BSDL %{buildroot}%{gem_instdir}/
+cp COPYING %{buildroot}%{gem_instdir}/
+
+%files
+%defattr(-,root,root,-)
+%license %{gemdir}/gems/%{gem_name}-%{version}/BSDL
+%license %{gemdir}/gems/%{gem_name}-%{version}/COPYING
+%{gemdir}
+
+%changelog
+* Thu Apr 21 2022 Neha Agarwal <nehaagarwal@microsoft.com> - 2.0.1-3
+- Adding patch to remove missing development_dependencies from .gemspec
```

Comment:
Double-checking: considering the patch, does the package still build correctly and provide working components?

Reply:
Yes, the package builds correctly in the pipelines